python/ray/experimental/state/state_cli.py

```diff
@@ -173,7 +173,15 @@ def output_with_format(
     if format == AvailableFormat.DEFAULT:
         return get_table_output(state_data, schema)
     if format == AvailableFormat.YAML:
-        return yaml.dump(state_data, indent=4, explicit_start=True)
+        augmented_state_data = [
+            {
+                ("task_id: " + state["task_id"])
+                if "task_id" in state
+                else "No task_id found": state
```

Comment:
we should always have task id - so this is probably not needed.

Reply:
sounds good! I can remove.